che-incubator/che-code
# chore: browser database name should contain a valid DevWorkspace identifier

### What does this PR do?
Uses a real workspace identifier taken from env.DEVWORKSPACE_ID to form the name of browser-based database.

### What issues does this PR fix?

https://github.com/eclipse/che/issues/22338

### How to test this PR?

- When you are on the dashboard, open browser development console, go to `Application`. In the `Storage` section below expand `IndexedDB` and remove all the databases
- Create a workspace with https://github.com/che-incubator/che-code/tree/vscode-web-state-db-identifier-test
- open a terminal, show a read `DEVWORKSPACE_ID`

```
$ export | grep DEVWORKSPACE_ID
```
- [1] ensure there is a database containing the ID at the end

- go to the dasboard, delete workspace, create a workspace using the same URI
- [2] check the workspace ID and check, that there is another database with this ID has been created

Browse URL must be the same for both workspaces.

![Screenshot from 2023-09-08 11-06-41](https://github.com/che-incubator/che-code/assets/1655894/3eb3df1a-5299-4bb5-b78b-3d8448146479)
